docs/Makefile

```diff
@@ -37,7 +37,41 @@ help:
 html-noplot:
 	$(SPHINXBUILD) -D plot_gallery=0 -b html $(SPHINXOPTS) "$(SOURCEDIR)" "$(BUILDDIR)/html"
 
-download:
+download-noplot:
+	mkdir -p $(GETTING_STARTED_IMAGES)
+
+	# download resources for s3 integration
+	wget -N https://storage.googleapis.com/datasets_boris/resources_s3_integration.zip -P $(DATADIR)
+	unzip $(ZIPOPTS) $(DATADIR)/resources_s3_integration.zip  -d $(GETTING_STARTED_IMAGES)
+
+	# download resources for azure integration
+	wget -N https://storage.googleapis.com/datasets_boris/resources_azure_integration.zip -P $(DATADIR)
+	unzip $(ZIPOPTS) $(DATADIR)/resources_azure_integration.zip -d $(GETTING_STARTED_IMAGES)
+	@if [ -d $(GETTING_STARTED_IMAGES)/resources ]; then \
+		mv $(GETTING_STARTED_IMAGES)/resources/* $(GETTING_STARTED_IMAGES)/;\
+		rm -R $(GETTING_STARTED_IMAGES)/resources/;\
+	fi
+
+	# download images and report for docker
+	wget -N https://storage.googleapis.com/datasets_boris/resources.zip -P $(DATADIR);\
+	unzip $(ZIPOPTS) $(DATADIR)/resources.zip  -d $(DOCKERSOURCE);\
+
+	# pizza dataset
+	@if [ ! -d $(PLATFORMSOURCE)/pizzas/salami ]; then \
```

Comment:
Why do you test for the salami subfolder and not the pizza folder?

Reply:
Because I want to make sure that the subfolder has already been created.